scalafix-core/shared/src/main/scala/scalafix/internal/rule/AsInstanceOfDisabled.scala

```diff
@@ -0,0 +1,31 @@
+package scalafix.internal.rule
+
+import scala.meta._
+import scalafix.rule.SemanticRule
+import scalafix.util.SemanticdbIndex
+import scalafix.rule.RuleCtx
+import scalafix.lint.LintMessage
+import scalafix.lint.LintCategory
+import scalafix.syntax._
+
+case class AsInstanceOfDisabled(index: SemanticdbIndex)
```

Comment:
What do you think about renaming this rewrite to `Disable` and make the disabled symbols configurable as is done with `NoInfer` in #355 ? Each rule does a full traversal on the tree so we should avoid a custom rule per disabled symbol.

Reply:
Hehe, I remember that @barambani wanted to do it this way, and then I convinced him to make it specific to `asInstanceOf` and if there's interest we can generalize it afterwards. 👍 I agree that if general, better!

Reply:
Fixed, thanks to both for the great feedback. Please, give it a look and let me know your thoughts.